internal/util/topology.go

```diff
@@ -30,6 +30,9 @@ import (
 const (
 	keySeparator   rune   = '/'
 	labelSeparator string = ","
+
+	// topologyPoolsParam is the parameter name used to pass topology constrained pools.
+	TopologyPoolsParam = "topologyConstrainedPools"
```

Comment:
this is not use outside of the package, so you can call it `topologyPoolsParam` as in the comment